### ISSUE TITLE
Invalid forms for payment methods have no error icon and no focus (accessibility)

### ISSUE THREAD

**Issue by M-Behr** [intershop/intershop-pwa#1791]
**Actual Behavior**

If credit card form or direct debit form is invalid, no error icon is visible. The focus is not nset to the first invalid field.

**Expected Behavior**

Error icon and focus behavior should be available for consistent behavior. It is important to avoid confusion and provide a more cohesive user experience.

**Steps to Reproduce the Bug**

1. Go to 'payment page' in the cart
2. Click on 'Add payment instrument'
3. Click on 'Submit'

**Additional Context, like Screenshots, Log File Snippets etc.**

![Image](https://github.com/user-attachments/assets/e8a69ce2-aeb8-42ae-a74e-873f2a2fd08d)

Expected:
![Image](https://github.com/user-attachments/assets/1cc43ddf-748e-4813-982d-cfafd0ff4369)

[AB#105812](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/105812)

**Comment by Timerk** [comment]
Just to clarify: this PR does not address issues with external payment providers that embed their forms in iframes. At the moment its not possible to inject our icons the these iframes.
